### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by KanekarSnehal**
### Overview
---
Created a static page for home with responsiveness.

Files to check

- [ ] src/components/Comment/Comment.jsx
- [ ] src/components/FollowMenuBar/FollowMenuBar.jsx
- [ ] src/components/MainContainer/MainContainer.jsx
- [ ] src/components/ModalInput/ModalInput.jsx
- [ ] src/components/PostList/PostList.jsx
- [ ] src/components/SideMenuBar/SideMenuBar.jsx
- [ ] src/components/SinglePost/SinglePost.jsx
- [ ] src/page/Home.jsx
- [ ] src/routes/AppRoutes.jsx

### CODE REVIEW

**Review by nerdyspook — APPROVED** [review]
Well structure of code maintained.

File: src/components/Comment/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import { MdDeleteOutline } from "react-icons/md";
+
+export const Comment = () => {
+  const [showOptions, setShowOptions] = useState(false);
+  return (
+    <>
+      <div className="mt-4 flex">
+        <img
+          className="h-8 rounded-full cursor-pointer"
+          src="https://res.cloudinary.com/dflebgpde/image/upload/v1652528175/socially/hacker_obpy2s.png"
```
Comment (shivanigangadharan) [inline src/components/Comment/Comment.jsx:13]: should provide an `alt` message for all `` tags

**Review by shivanigangadharan — APPROVED** [review]
Code looks clean, UI is really good with responsiveness too. 👍🏼

**Review by sharathnair9999 — APPROVED** [review]

File: src/components/Comment/Comment.jsx (hunk 1 above)
Comment (KanekarSnehal) [reply src/components/Comment/Comment.jsx:13]: done

**Pull request merged**